app/models/item.rb

```diff
@@ -1,6 +1,5 @@
 class Item < ApplicationRecord
   validates_presence_of :name,
-                       :active,
```

Comment:
Why did we remove this? It should never be null.

Reply:
Added assertion / line in model for `validates_inclusion_of` with true and false